README.md

```diff
@@ -51,8 +51,10 @@ $ yarn add @digipolis/authz
 ### Available sources:
 #### authzv2:
 For applications which use the User Management Engine and have a JWT token of the authenticated user (mostly API's).
+Authzv2 works with jwt tokens. You can use the jwt-up policy to convert it
```

Comment:
Is het niet beter: "convert it" -> "to construct a jwt token for the upstream service"

Reply:
aangepast